### ISSUE TITLE
As a selector, I would like to review the non-Reserves CDL items so I can decide whether or not to reshelve them

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#4314]
Since having an item on CDL makes it unavailable for checkout or in-Library use, selectors would like to be able to decide whether to reshelve items. For reserves items, we already have a process for doing that at the end of the semester, but for non-Reserves items, they will remain on CDL indefinitely unless we have a mechanism to review and reshelve items.

**Comment by hackartisan** [comment]
Figgy doesn't know whether something is CDL until it asks voyager, and I don't think it knows whether something is reserve or not ever. Perhaps a voyager report makes sense here?

**Comment by kelea99** [comment]
arent all CDL items tagged as the CDL collection in figgy? Selectors could go through and look at the collection itself? but no, it is not known whether or not it is a reserves item in figgy. I wonder if those should be tagged as such.

**Comment by escowles** [comment]
Yes, this should probably be a marc_liberation ticket — selectors could review the items in Figgy, but they would have no way of knowing whether the item was a reserves item, or even whether it had been reshelved.
